docs/getting-started/installation-and-upgrade/installation-references/helm-chart-options.md

```diff
@@ -31,7 +31,7 @@ For information on enabling experimental features, refer to [this page.](../../.
 | Option                         | Default Value                                         | Description                                                                                                                                       |
 | ------------------------------ | ----------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `additionalTrustedCAs`         | false                                                 | `bool` - See [Additional Trusted CAs](#additional-trusted-cas)                                                                                    |
-| `addLocal`                     | "true"                                                | `string` - Have Rancher detect and import the "local" Rancher server cluster.  _Note: This option is no longer available in v2.5.0. Consider using the `restrictedAdmin` option to prevent users from modifying the local cluster._ |
+| `addLocal`                     | "true"                                                | `string` - Have Rancher detect and import the "local" Rancher server cluster.  _Note: This option is no longer available in v2.5.0.               |
```

Comment:
```suggestion
| `addLocal`                     | "true"                                                | `string` - Have Rancher detect and import the "local" (upstream) Rancher server cluster.  _Note: This option is no longer available in v2.5.0.               |
```

Reply:
Can you elaborate on why you would add this phrase? Given that upstream has other meanings (i.e. not owned by us, for example something like kubernetes is an upstream resource) I don't think that this helps clarify what the local cluster is, but if we use upstream to refer to the local cluster elsewhere I can make this clarification.

Reply:
Sounds good, I've made the requested change.